lib/appoptics_apm/frameworks/rails/inst/active_record.rb

```diff
@@ -1,21 +1,20 @@
 # Copyright (c) 2016 SolarWinds, LLC.
 # All rights reserved.
 
-require 'appoptics_apm/frameworks/rails/inst/connection_adapters/mysql'
 require 'appoptics_apm/frameworks/rails/inst/connection_adapters/mysql2'
```

Comment:
Does this work with either/or `mysql` and `mysql2` or is it now only `mysql2`?

Reply:
only mysql2 , mysql is EOL